__tests__/api.test.js

```diff
@@ -241,4 +270,129 @@ describe("PATCH /api/articles/:article_id/", () => {
         expect(res.body.msg).toBe("bad request");
       });
   });
+
+  test("PATCH:400 for invalid article ID format", () => {
+    return request(app)
+      .patch("/api/articles/not-a-number")
+      .send({ inc_votes: 1 })
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("bad request");
+      });
+  });
+});
+
```

Comment:
You could have another 400 test here for : 

 - invalid data type for inc_vote value..e.g - {inc_vote: "cheese"}